README.md

```diff
@@ -62,6 +63,11 @@ A curated list of notable ETL (extract, transform, load) frameworks, libraries a
 ## Go
 * [Crunch](https://github.com/jondot/crunch) - "A fast to develop, fast to run, Go based toolkit for ETL and feature extraction on Hadoop."
 
+## Salesforce
+* [Data Loader](https://developer.salesforce.com/page/Data_Loader) - "is an easy to use graphical tool that helps you to get your data into Salesforce objects."
+* [SFXOrgData](https://www.sfapex.com/) - "copy data from your Production or Full Sandbox environment to Developer/Dev Pro sandboxes."
```

Comment:
It's realy salesforce specific, but maybe it's apropriate for Salesforce section? Do you think, I should remove SFXOrgData from the list?